py3status/modules/google_calendar.py

```diff
@@ -59,13 +60,13 @@
     warn_timeout: The number of seconds before a warning should be displayed again.
         (default 300)
 
-format placeholders:
+Format placeholders:
     {events} All the events to display.
 
-format_error placeholders:
+Format_error placeholders:
     {error} Error to display.
```

Comment:
We should play with this eventually as it could not be needed here.

Reply:
Yeah, I need an idea of how to handle this. The only real case is when the module starts up without a connection. We could possibly just not display anything in that case, or a loading symbol of some sort like your `gcal` module. Also, the time_out without a connection is hard-coded to `5`, if you remember, so maybe we make that a parameter?